bin/suitcss

```diff
@@ -63,9 +63,16 @@ program.parse(process.argv);
  * Settings.
  */
 
+function requireOrParse(configFile) {
+  var configPath = resolve(configFile);
+  var ext = path.extname(configPath);
+  var readFn = ext === '.js' ? require : fs.readJsonSync;
```

Comment:
if @simonsmith thinks that it is not a big deal to require json with `readJsonSync` (it could be `require`d) then we can merge this.